docs/conf.py

```
@@ -312,8 +467,80 @@
+
+# try to link with official python3 documentation.
+# see https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html for more
+intersphinx_mapping = sanitize_intersphinx_mapping(
+	{
+		"python": ("https://docs.python.org/3", (None, "python-inv.txt")),
+		"PEP": ("https://peps.python.org", (None, "pep-inv.txt")),
+	},
+)
+
+
+def linkcode_resolve(domain: Any, info: Any) -> str:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)